app/assets/stylesheets/pages/_events_show.scss

```diff
@@ -0,0 +1,55 @@
+
+.logo {
+text-align: center;
+font-weight: 300;
+font-size: 35px;
+}
+
+.title1 {
+  text-align: center;
+  top: 40px;
+  font-weight: bolder;
+  margin-top: 100px;
+  font-size: 30px;
+}
+
+.button1 {
+  vertical-align: center;
+}
+
+.card-center {
+  text-align: -webkit-center;
+}
+
+.small-text {
+  font-size: 15px;
+  text-align: top;
+}
+
+.add {
+
+  background-color: #DBAD6A;
+  border: none;
+  color: white;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  border-radius: 15px;
+  margin-bottom: 4px;
+}
+
+.container.falafel {
```

Comment:
no falafel what the fuck...

Reply:
Not me as we spoke about yesterday.

Reply:
ok, please adjust regardless so we don't lose time over this